src/stores/models/posting/Upload.js

```diff
@@ -36,7 +36,14 @@ export default Upload = types.model('Upload', {
 		},
 
 		copy_markdown_to_clipboard() {
-			let markdown = `![](${ self.url })`
+			let markdown = ""
+			if (self.alt && self.alt.length > 0) {
+				markdown = `![${ self.alt.replace('"', '') }](${ self.url })`
```

Comment:
```suggestion
				markdown = `![${ self.alt }](${ self.url })`
```

And because it's markdown, `.replace('"', '')` could be removed. But to be on the safe side, I left it there.